.github/workflows/business-pay-ocp-cd.yml

```diff
@@ -24,7 +24,7 @@ env:
 
 jobs:
   entity-pay-cd-by-push:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
I decided to go with v24.04 because that's what SBC Connect Team did for the Auth CI scripts.